sdk/search/azure-search-documents/tests/search_service_preparer.py

```diff
@@ -138,20 +141,21 @@ def decorator(func):
         def wrapper(*args, **kwargs):
             # set up hotels search index
             test = args[0]
-            api_key = kwargs.get("search_service_api_key")
+            # api_key = kwargs.get("search_service_api_key")
             endpoint = kwargs.get("search_service_endpoint")
             service_name = kwargs.get("search_service_name")
             if test.is_live:
-                _clean_up_indexes(endpoint, api_key)
-                _set_up_index(service_name, endpoint, api_key, schema, index_batch)
-                _clean_up_indexers(endpoint, api_key)
+                cred = get_credential()
+                _clean_up_indexes(endpoint, cred)
+                _set_up_index(service_name, endpoint, cred, schema, index_batch)
+                _clean_up_indexers(endpoint, cred)
             index_name = json.loads(_load_schema(schema))["name"] if schema else None
             index_batch_data = _load_batch(index_batch) if index_batch else None
 
             # ensure that the names in the test signatures are in the
             # bag of kwargs
             kwargs["endpoint"] = endpoint
-            kwargs["api_key"] = AzureKeyCredential(api_key)
+            # kwargs["api_key"] = AzureKeyCredential(api_key)
             kwargs["index_name"] = index_name
             kwargs["index_batch"] = index_batch_data
 
```

Comment:
Below here on line 172 there is a `# FIXME: DELETE EVERYTHING AFTER THIS LINE BEFORE MERGING`. Should we use this as an opportunity to do so?

Reply:
I plan to use this PR to show our partner teams what changes are needed for test migration. I don't want to include those changes which may cause confusion. Will do that in a separate change. (the header change is small enough in my mind, but I am open if you prefer moving it out of the PR)

Reply:
Yea, don't mind either way. I see that you went ahead and removed it in any case. Should be easy enough to see why you deleted code after a line that says "DELETE EVERYTHING AFTER THIS LINE" without causing confusion haha.